README.md

```diff
@@ -1,44 +1,180 @@
 Ballerina AWS Redshift Connector
```

Comment:
Would it be possible to add the version number and little bit of info API that we use to build this connector in the overview section (since that package.md will removed in the future)?

Reply:
e054c610291a4ff59d7f321102fa99158f3489ef